DiscordBot/README.md

```diff
@@ -0,0 +1,43 @@
+# Creating a Discord Bot in Python
+
+This lesson covers the programming concepts required to build a Discord bot in Python, including importing a 3rd party library, reading API documentation, and using decorators. The goal is to have a simple, working discord bot that students can add to a Discord server.
+
+## Warm-Up Activity
+
+Present the [Chat Bot Introduction](ChatBotIntro.pptx) PowerPoint Presentation. At the end, have the students join the [Chat Bot Sandbox](https://discord.gg/7VUAKkHEvm) Discord server. If they do not have an account, walk them through the account creation process. When students have joined the server, grant them the Bot Tester role. This will allow them to add their Discord bots to the server once they are created.
+
+## PowerPoint Presentation
+
+Present the [Imports, Arguments, and Decorators](ImportsArgsAndDecorators.pptx) PowerPoint Presentation. Review the presentation ahead of time, including the notes in the slides.
+
+## Group Activity
+
+For the group activity, facilitate the [Decorators and Collections Review Kahoot](https://create.kahoot.it/details/7042ea04-fe2a-4252-b48b-de3b63f583be).
+
+1. Follow the above link to the Kahoot quiz
+1. Select the "Play" button, then select "Host" from the modal
+1. Select "Classic" mode
+1. Share the play.kahoot.it link and the Kahoot PIN in chat with the students
+    - Award points based on the final standings:
+        - 1 point for participation
+        - 2 extra points for 1st place
+        - 1 extra point to 2nd and 3rd place
+
+## Code-Along Activity
+
+The [activity](DiscordCodeAlong.md) walks through the creation of a Discord bot, including registering the bot as a Discord application, adding the bot to the server, making the bot respond to messages, and adding a simple command to the bot. This will use the concepts introduced in the PowerPoint, including using 3rd Party APIs and decorators.
```

Comment:
```suggestion
The [activity](DiscordCodeAlong.md) walks through the creation of a Discord bot, including registering the bot as a Discord application, adding the bot to the server, and making the bot respond to messages. This will use the concepts introduced in the PowerPoint, including using 3rd Party APIs and decorators.
```

Reply:
👍